manifests/record/a.pp

```diff
@@ -8,21 +8,32 @@
   $data,
   $ttl = '',
   $ptr = false,
-  $host = $name ) {
+  $host = $name,
+  $data_dir = $::dns::server::params::data_dir,
```

Comment:
@solarkennedy I can make the change, but every other parameterized class and reference to a parameter in this module is done directly with `dns::server::params`. It would be nice to have them switched to `dns::server::config` as well.